### PR TITLE
Add syntax highlight for pig-latin

### PR DESCRIPTION
**Pull request by zjffdu**
I use the syntax template for sublime and convert it into ace js via instruction here. https://github.com/ajaxorg/ace/wiki/Importing-.tmtheme-and-.tmlanguage-Files-into-Ace
And also make a minor change after the automatic generation. I have verified it in kitchen-sink.html

* https://github.com/chrislongo/Pig

Screenshot of pig sample code

image

### CODE REVIEW

**Comment by zjffdu** [comment]
Is anyone still maintain this project ?

**Comment by nightwing** [comment]
Sorry for the delay. We had to update our CLA, and until it is done we can't merge non-trivial pull requests. It should be ready next week, i'll send you the CLA to sign once it is ready.

Again sorry for the late reply, and thanks for the contribution.

File: lib/ace/mode/pig.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+var Mode = function() {
+    this.HighlightRules = PigHighlightRules;
+    this.foldingRules = new FoldMode();
+};
+oop.inherits(Mode, TextMode);
+
+(function() {
+    // this.lineCommentStart = ""--.*$"";
+    // this.blockComment = {start: ""/*"", end: ""*/""};
```
Comment (nightwing) [inline lib/ace/mode/pig.js]: these lines need to be changed to 
```js
    this.lineCommentStart = "--";
    this.blockComment = {start: "/*", end: "*/"};
```
for commenting to work.

Comment (zjffdu) [inline lib/ace/mode/pig.js]: @nightwing It doesn't work after I make this change. Actually comment highlight already works, I guess it is specified in line 49 of file `pig_highlight_rules.js`

**Comment by zjffdu** [comment]
@nightwing Thanks for your review.

File: lib/ace/mode/pig.js (hunk 1 above)
Comment (nightwing) [inline lib/ace/mode/pig.js]: This is needed for cmd-/ (togglecomment command).
What errors do you see after adding these lines?

Comment (zjffdu) [inline lib/ace/mode/pig.js]: Sorry, my mistake, it's working now.

**Comment by zjffdu** [comment]
ping @nightwing

**Comment by nightwing** [comment]
Could you please confirm once more in a comment here that this contribution is under the terms of the project's BSD license. I can merge after that, thanks.

**Comment by zjffdu** [comment]
I confirm this PR is under the terms of the project's BSD license.

**Comment by zjffdu** [comment]
Thanks @nightwing,  May I know when the next version will be released so that I use it in my project ?

**Comment by nightwing** [comment]
Most likely at the end of april, but may be sooner if we manage to fix IME issues quickly.

**Comment by zjffdu** [comment]
@nightwing Any plan for release ? Thanks

**Comment by zjffdu** [comment]
ping @nightwing

**Comment by zjffdu** [comment]
ping @nightwing Any news for release ? Thanks

**Comment by zjffdu** [comment]
Sorry, just notice that there's one new release several days ago.

**Comment by nightwing** [comment]
it is in https://github.com/ajaxorg/ace-builds/releases/tag/v1.2.7

**Pull request merged**